calendarsubscription/calendarsubscription.go

```diff
@@ -46,3 +50,45 @@ func (cs CalendarSubscription) Normalize() (*CalendarSubscription, error) {
 
 	return &cs, nil
 }
+
+func (cs CalendarSubscription) renderICalFromShifts(shifts []oncall.Shift, reminderMinutes []int) ([]byte, error) {
+	type iCalOptions struct {
+		Shifts          []oncall.Shift `json:"s,omitempty"`
+		ReminderMinutes []int          `json:"r,omitempty"`
+	}
+
+	var iCalTemplate = `
+		BEGIN:VCALENDAR
```

Comment:
We should add a comment above this template to the RFC it's built against, I'm assuming this one: https://tools.ietf.org/html/rfc5545